README.md

```diff
@@ -122,7 +123,7 @@ happened with vcs-sync.
 ```
 curl: (22) The requested URL returned error: 404 Not Found
 parallel: This job failed:
-curl -SsfL --compressed https://index.taskcluster.net/v1/task/gecko.v2.mozilla-beta.revision.c76e2781238741c8c9822725da3dffc2d96c282c.firefox.win64-searchfox-debug/artifacts/public/build/target.mozsearch-index.zip > win64.mozsearch-index.zip
+curl -SsfL --compressed https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/gecko.v2.mozilla-beta.revision.c76e2781238741c8c9822725da3dffc2d96c282c.firefox.win64-searchfox-debug/artifacts/public/build/target.mozsearch-index.zip > win64.mozsearch-index.zip
```

Comment:
my one comment, this url won't likely be found, since the rev referenced was built on the old cluster. But I suspect updating the rev wouldn't be difficult.